### PR TITLE
Create Zig.gitignore

### PR DESCRIPTION
**Pull request by el10savio**
Added Zig specific .gitignore
**Reasons for making this change:**

_TODO_

**Links to documentation supporting these rule changes:**

_TODO_

If this is a new template:

 - **Link to application or project’s homepage**: _TODO_

### CODE REVIEW

File: Zig.gitignore [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+# Cheers!
+# -andrewrk
+
+zig-cache/
+zig-out/
+/release/
+/debug/
+/build/
+/build-*/
+/docgen_tmp/
```
Comment (g-w1) [inline Zig.gitignore:18]: only zig-cache and zig-out are needed. The rest are specific to the zig repository, and not every zig project.

File: Zig.gitignore [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# This file is for zig-specific build artifacts.
+# If you have OS-specific or editor-specific files to ignore,
+# such as *.swp or .DS_Store, put those in your global
+# ~/.gitignore and put this in your ~/.gitconfig:
+#
+# [core]
+#     excludesfile = ~/.gitignore
+#
+# Cheers!
+# -andrewrk
```
Comment (g-w1) [inline Zig.gitignore:10]: Not needed.

**Review by g-w1 — CHANGES_REQUESTED** [review]

File: Zig.gitignore (hunk 1 above)
Comment (el10savio) [reply Zig.gitignore]: https://github.com/el10savio/gitignore/commit/08f8a14aeefbc399c6a53878a5bf45ee9e3439c4

File: Zig.gitignore (hunk 2 above)
Comment (el10savio) [reply Zig.gitignore]: https://github.com/el10savio/gitignore/commit/08f8a14aeefbc399c6a53878a5bf45ee9e3439c4

**Review by g-w1 — APPROVED** [review]

**Comment by bdougie** [comment]
Thanks for the PR, please provide more context on why this is needed. See the project's [README](https://github.com/github/gitignore#readme) for help.

**Pull request closed without merging**